cli/integration_tests/edit_turbo_json/global.t

```diff
@@ -0,0 +1,32 @@
+Setup
+  $ . ${TESTDIR}/../_helpers/setup.sh
+  $ . ${TESTDIR}/../_helpers/setup_monorepo.sh $(pwd)
+
+Baseline global hash
+  $ cp "$TESTDIR/fixture-configs/1-baseline.json" "$(pwd)/turbo.json" && git commit -am "no comment" --quiet
+  $ BASELINE=$(${TURBO} build -vv 2>&1 | "$TESTDIR/../_helpers/get-global-hash.sh")
+
+Update pipeline: global hash remains stable.
+  $ cp "$TESTDIR/fixture-configs/2-update-pipeline.json" "$(pwd)/turbo.json" && git commit -am "no comment" --quiet
```

Comment:
It's pretty hard to see the diff between these configs. Could you model them as git patches or do it inline with `jq` so it's easieer to see what's changing and then be able to validate that the test is doing the right thing also?

Reply:
I just attempted this in two different ways. Neither of them are a better overall experience.

***

### Either:
Can introduce transitive failures in testing. After my first attempt with diffs things just _didn't work_. Where did this problem get introduced? But since it's not on disk at the point at which we're debugging the experience is miserable.

### Diffs:
The test authoring experience for this becomes "resolve all the files again at each step" and then recalculate the diffs. That's not good.

### Programmatic:
We're in a shell environment; `jq` to set multiple values on a change results in basically inlining a bunch of assignments.

For example:
`jq -r '.pipeline.build.env = ["CC"] | .passThroughEnv="SOMETHING"'` and this keeps getting longer and longer the more changes that need to be made.

***

Open them in your editor and toggle between them if you need to compare.

Reply:
Easy peasy:
<img width="923" alt="comparing two fixture files" src="https://user-images.githubusercontent.com/20542/235145123-91772c2d-dd4d-4dba-853a-a42d8a99204d.png">